aspnetcore/tutorials/signalr.md

```diff
@@ -90,76 +87,85 @@ At the end, you'll have a working chat app:
 
 ## Add the SignalR client library
 
-The SignalR server library is included in the [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app). But you have to get the JavaScript client library from [npm, the Node.js package manager](https://www.npmjs.com/get-npm).
+The SignalR server library is included in the [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app). The JavaScript client library isn't automatically included in the project. For this tutorial, you use the [Library Manager (LibMan)](xref:client-side/libman/index) to get the client library from *unpkg*. [unpkg](https://unpkg.com/#/) is a [content delivery network](https://wikipedia.org/wiki/Content_delivery_network) that can deliver anything found in [npm, the Node.js package manager](https://www.npmjs.com/get-npm).
 
 # [Visual Studio](#tab/visual-studio/)
 
-* In **Package Manager Console** (PMC), change to the project folder (the one that contains the *SignalRChat.csproj* file).
+* In **Solution Explorer**, right-click the project, and select **Add** > **Client-Side Library**.
 
-  ```console
-  cd SignalRChat
-  ```
+* In the **Add Client-Side Library** dialog, for **Provider** select **unpkg**. 
 
-# [Visual Studio Code](#tab/visual-studio-code/)
+* For **Library**, enter _@aspnet/signalr@1_, and select the latest version that isn't preview.
 
-2. Change to the new project folder.
+  ![Add Client-Side Library dialog - select library](signalr/_static/libman1.png)
 
-  ```console
-  cd SignalRChat
-  ``` 
+* Select **Choose specific files**, expand the *dist/browser* folder, and select *signalr.js* and *signalr.min.js*.
 
-# [Visual Studio for Mac](#tab/visual-studio-mac)
+* Set **Target Location** to *wwwroot/lib/signalr/*, and select **Install**.
```

Comment:
select --> click

Reply:
On select vs. click, the style guide recommends avoiding click:

> Customers interact with products using different input methods: keyboard, mouse, touch, voice, and more. So use generic verbs that work with any input method. Don’t use input-specific verbs, such as click or swipe. Instead, use the verbs listed here. 

Usually that means use select or choose instead of click. Though sometimes that doesn't work -- when a button has the label "Select" I say "click Select", not "select Select".

Incidentally, I noticed in this same style guide article a note about the menu > menu format:  

> Don’t bold the greater-than symbol. Include a space before and after the symbol.

😄 

https://worldready.cloudapp.net/Styleguide/Read?id=2700&topicid=26472